azurerm/resource_arm_api_management_api.go

```diff
@@ -243,6 +243,9 @@ func resourceArmApiManagementApiCreateUpdate(d *schema.ResourceData, meta interf
 	displayName := d.Get("display_name").(string)
 	serviceUrl := d.Get("service_url").(string)
 
+	version := d.Get("version").(string)
+	versionSetId := d.Get("version_set_id").(string)
```

Comment:
we also need to set this field back into the state in the Read function below (`resourceArmApiManagementApiRead`) and [add this to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/api_management_api.html.markdown#L67) - could we also add an Acceptance Test for this ([example](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/azurerm/resource_arm_api_management_api_test.go#L14-L40))? this'd involve adding a new test setting this field to confirm this works - which can be [run using the instructions in the README](https://github.com/terraform-providers/terraform-provider-azurerm#developing-the-provider)

Reply:
I put in a first pass at setting the Read state, but I'm not sure how to validate. Do you have a way to debug with breakpoints?

Reply:
@tombuildsstuff, I believe I have these items covered now.